bootstrap-extensions/pom.xml

```diff
@@ -21,7 +21,7 @@
         <spin-js.version>2.1.0</spin-js.version>
         <animate.css.version>3.7.2</animate.css.version>
         <summernote.version>0.8.20</summernote.version>
-        <font-awesome.version>5.15.4</font-awesome.version>
+        <font-awesome.version>6.1.0</font-awesome.version>
```

Comment:
we should ping someone to release `6.1.1` :)

Reply:
yes, of course. https://github.com/FortAwesome/Font-Awesome/issues/19092

Reply:
font-awesome-6.1.1 is available in maven central now.